website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+// All of the following configs are used in the website and documentation provided by the Design team Glyph
+
+// Colors
+
+@mixin detox-colors($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
+  --#{$prefix}-y0: #f7ba29ff;
+  --#{$prefix}-b0: #3e77e4ff;
+  --#{$prefix}-u0: #7633e5ff;
+  --#{$prefix}-e0: #3ebf72ff;
+
+  --#{$prefix}-r1a: #f14c5289;
+  --#{$prefix}-y1a: #f7ba2989;
+  --#{$prefix}-b1a: #3e77e489;
+  --#{$prefix}-u1a: #7633e589;
+  --#{$prefix}-e1a: #3ebf7289;
+
+  --#{$prefix}-r2a: #f14c5259;
+  --#{$prefix}-y2a: #f7ba2959;
+  --#{$prefix}-b2a: #3e77e459;
+  --#{$prefix}-u2a: #7633e559;
+  --#{$prefix}-e2a: #3ebf7259;
+
+  --#{$prefix}-r3a: #f14c5240;
+  --#{$prefix}-y3a: #f7ba2940;
+  --#{$prefix}-b3a: #3e77e440;
+  --#{$prefix}-u3a: #7633e540;
+  --#{$prefix}-e3a: #3ebf7240;
+
+  --#{$prefix}-r4a: #f14c5226;
+  --#{$prefix}-y4a: #f7ba2926;
+  --#{$prefix}-b4a: #3e77e426;
+  --#{$prefix}-u4a: #7633e526;
+  --#{$prefix}-e4a: #3ebf7226;
+
+  --#{$prefix}-r5a: #f14c5212;
+  --#{$prefix}-y5a: #f7ba2912;
+  --#{$prefix}-b5a: #3e77e412;
+  --#{$prefix}-u5a: #7633e512;
+  --#{$prefix}-e5a: #3ebf7212;
+
+  --#{$prefix}-a0: #12161bff;
+  --#{$prefix}-a1: #babbbdff;
+  --#{$prefix}-a2: #e0e1e1ff;
+  --#{$prefix}-a3: #f1f1f1ff;
+  --#{$prefix}-a4: #fafafaff;
+  --#{$prefix}-a5: #ffffffff;
+
+  --#{$prefix}-p0: #5c25afff;
+  --#{$prefix}-p1: #8157c1ff;
+  --#{$prefix}-p2: #9876cdff;
+  --#{$prefix}-p3: #a789d4ff;
+  --#{$prefix}-p4: #b9a1ddff;
+  --#{$prefix}-p5: #d3c4e9ff;
+}
+
+@mixin ifm-colors($prefix) {
+  --#{$prefix}-color-primary: #048fc0;
+  --#{$prefix}-color-primary-dark: #253282;
+  --#{$prefix}-color-primary-darker: #161360;
+  --#{$prefix}-color-primary-darkest: #150a51;
+  --#{$prefix}-color-primary-light: #6c43a3;
+  --#{$prefix}-color-primary-lighter: #8859c9;
+  --#{$prefix}-color-primary-lightest: #9f6de5;
+  --#{$prefix}-code-font-size: 95%;
+  --#{$prefix}-highlighted-code-line-bg: #d9e5f6;
+}
+
+@mixin detox-colors-dark($prefix) {
+  --#{$prefix}-highlighted-code-line-bg: #414458;
+  --ifm-background-color: #12161b;
+
+  --#{$prefix}-color-a0: #ffffff;
+  --#{$prefix}-color-a1: #12161b;
+  --#{$prefix}-color-a2: #171b20;
+  --#{$prefix}-color-a3: #202429;
+  --#{$prefix}-color-a4: #313439;
+  --#{$prefix}-color-a5: #575a5d;
+}
+
+:root {
+  @include detox-colors('detox-color');
+  @include ifm-colors('ifm');
+
+  &[data-theme='dark'] {
+    @include detox-colors-dark('detox-color');
+  }
+}
+
+// Fonts
+
+@mixin h1-80 {
+  font-family: 'Nunito Sans';
```

Comment:
Looks not bad. Here I agree that mixins make sense! We are likely to use them multiple times.

What I like less is that you define only a few properties. That seems very risky - what if there are some previously defined things like: `font-style: italic` etc.?

I'd vote for defining `font` shorthands in such mixins. That should be a safer option:

```
font: 400 normal 80px/96px 'Nunito Sans', sans-serif.
```

Reply:
Au fait, you haven't defined line-heights in the mixins. Looks very risky 😬

Reply:
@noomorph - using the font shorthand property is a 😎 cool idea! You can put a few values in there. When designers specify the font size and line-height in the format "24px / 30px", it means that the font size is 24 pixels and the line-height is 30 pixels. So the line-heights are there. and no need to specify "normal" since 400 is a regular/normal font weight.

Is this correct? If so, I have adjusted the fonts mixins to follow your suggestion and added only the color property of white

Reply:
Cool